### DISCUSSION TITLE
support crawling all websites in the specified country and skip subdirectories

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/katana#179] [Q&A] [answered]
I want to crawl all websites with different domains, limited regions (such as Thailand), and unlimited depth. In short, I want to crawl all websites in the specified country, but skip subdirectories of the same website category

It is mainly used to collect basic information of all websites in specific countries

**Comment by ehsandeep** [comment, marked as answer]
@hktalent you can use crawl scope option for this, for example:

```
cat urls.txt | katana -cs /th/
```

You can adopt the keyword, regex as you want.

**Reply by hktalent** [reply]
It can't be fully satisfied. You know, there are many IP designed url in the webpage

similar: http://33.3.79.1:9080/xxx.login
